### DISCUSSION TITLE
Caso d'uso Trascr_020: flag richiedente madre/intestatario

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#941] [Supporto servizi cooperativi]
Nel dettaglio relativo al caso d'uso in oggetto è riportata, in fondo, la sezione:

![image](https://github.com/italia/ansc/assets/26415430/72470735-a774-4d45-99c7-8e7401158906)

che ci pare debba essere rimossa dal momento l'indicazione del flag dichiarante/comparente/firma è già presente sia nella sezione "madre" che nella sezione "Intestatario".

**Comment by asqueglia** [comment]
Buongiorno la segnalazione è stata corretta con il rilascio odierno della versione 1.30.0
